TestAll.ps1

```diff
@@ -0,0 +1,76 @@
+<#  
+.SYNOPSIS
+    Run the TAEF dlls generated by WinAppSDK
+
+.DESCRIPTION
+    The TestAll script will take the folder input and look for subfolders containing a .testmd file. WinAppSDK
```

Comment:
RECOMMENDATION: Pick a unique extension

This is similar-but-different from `*.testmd` Windows has (for some time). Please pick a different (unique) extension to avoid confusion.

SUGGESTIONS: `.testdef` (TEST DEFinition) but yeah, my namingFu feels underpowered atm so suggestions welcome

Reply:
+1 for using a different extension to avoid confusion with the existing testmd, ".testConfig" and ".testProp" are also options.

Reply:
Using testdef